server/server.js

```diff
@@ -0,0 +1,13 @@
+const http = require('http')
+const port = 3000
+const requestHandler = (request, response) => {
+    console.log(request.url)
+    response.end('Hello Node.js Server!')
+}
+const server = http.createServer(requestHandler)
+server.listen(port, (err) => {
+    if (err) {
+        return console.log('something bad happened', err)
```

Comment:
No need to return `console.log` result. It would be better and more readable to do

~~~js
console.log('something bad happened', err)
return
~~~